builtin/providers/aws/resource_aws_sns_topic.go

```diff
@@ -183,9 +185,14 @@ func resourceAwsSnsTopicRead(d *schema.ResourceData, meta interface{}) error {
 				// Some of the fetched attributes are stateful properties such as
 				// the number of subscriptions, the owner, etc. skip those
 				if resource.Schema[iKey] != nil {
-					value := *attrmap[oKey]
+					var value string
+					if iKey == "policy" {
```

Comment:
Hi @keymon 

This if statement feels redundant - we are doing the *same* thing in the if & else - is this actually the case?